src/test/java/io/jenkins/blueocean/autofavorite/FavoritingScmListenerTest.java

```diff
@@ -95,16 +109,22 @@ private WorkflowJob createAndRunPipeline() throws java.io.IOException, Interrupt
 
```

Comment:
The tests are failing here, which seems unrelated to what I changed, and they all pass if I run them from my desk.  Any ideas?